### PR TITLE
Fix compilation error in Xcode 11 Beta 2

### PR DESCRIPTION
**Pull request by antranapp**

### CODE REVIEW

File: SwiftUITodo/TaskListView.swift [hunk 1]
```diff
@@ -31,6 +31,7 @@ struct TaskListView: View {
   }
 
   private func createTask() {
+    print("task created")
```
Comment (devxoul) [inline SwiftUITodo/TaskListView.swift]: Can you get rid of this line?

Comment (antranapp) [reply SwiftUITodo/TaskListView.swift]: Oops, removed. just wanted to debug why I can't add a new task. It seems that the `onCommit` doesn't get called in Xcode 11 Beta 2. I haven't tested on real device though.

**Comment by devxoul** [comment]
@peacemoon Thanks for your contribution!

**Comment by antranapp** [comment]
One question not related to the PR, which tool do you use to generate the screenshot? Cmd+S from simulator only captures the content of the Simulator. I also want to have the iPhone frame but don't know how to do that.

**Comment by devxoul** [comment]
@peacemoon

1. Press <kbd>Command</kbd> + <kbd>Shift</kbd> + <kbd>4</kbd>
2. Move the cursor on the Simulator
3. Press <kbd>Space</kbd>
4. Click the Simulator
5. Check the screenshot file on Desktop

**Pull request merged**